engine/xml-config/src/test/java/org/datacleaner/configuration/DomConfigurationWriterTest.java

```diff
@@ -404,6 +405,44 @@ public void testWriteAndReadAllStringPatterns() throws Exception {
         assertEquals(false, regexStringPattern.isMatchEntireString());
     }
 
+    @Test
+    public void testWriteAndReadFixedWidthDatastore() throws Exception {
+        final FileResource fileResource = new FileResource("C://test.csv");
```

Comment:
Do not make assumptions about the file system. How would this test ever pass in a non-Windows environment?